index.html

```diff
@@ -1,182 +1,185 @@
+                <img src="images/Rectangle 19.png" alt="rectangle19" class="rectangle19">
+            </ul>
+            <ul class="skills">
+                <li id="title">Skills</li>
+                <li>Database Management</li>
+                <li>CLI</li>
+                <li>API Design</li>
+                <li>Version Control</li>
+                <li>Web Development</li>
+                <img src="images/light.png" alt="light-bulb" class="bulb">
+            </ul>
 
-          <div class="get-contact">
-            <img src="images/Shape (1).png" alt="image4" class="image4">
-            <img src="images/Shape (2).png" alt="image5" class="image5">
-            <h1>Get Started</h1>
-            <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch</p>
-            <form class="contact-form">
-            <div>
-              <label id="name-label">Your name</label>
-              <br>
-              <input id="name"  type="text" placeholder="Ricardo"/>
-            </div>
-            <div>
-              <label id="email-label">Your email</label>
-              <br>
-              <input id="email" type="email" placeholder="rj_egas@..."/>
-            </div>
-            <div>
-              <label>Your message</label>
-              <br>
-              <textarea name="comments" id="comment-box" cols="30" rows="10" placeholder="Enter your message here..."></textarea>
+            <div class="get-contact">
+                <img src="images/Shape (1).png" alt="image4" class="image4">
+                <img src="images/Shape (2).png" alt="image5" class="image5">
+                <h1>Get Started</h1>
+                <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch</p>
+                <form class="contact-form">
+                    <div>
+                        <label id="name-label">Your name</label>
+                        <br>
+                        <input id="name" type="text" placeholder="Ricardo" />
+                    </div>
+                    <div>
+                        <label id="email-label">Your email</label>
+                        <br>
+                        <input id="email" type="email" placeholder="rj_egas@..." />
+                    </div>
+                    <div>
+                        <label>Your message</label>
+                        <br>
+                        <textarea name="comments" id="comment-box" cols="30" rows="10"
+                            placeholder="Enter your message here..."></textarea>
+                    </div>
+                    <div>
+                        <button type="button" class="Collaboration">Start Collaboration</button>
+                    </div>
             </div>
-            <div>
-              <button type="button" class="Collaboration">Start Collaboration</button>
-            </div>
-         </div>
-     </div>
+        </div>
     </section>
     <footer>
```

Comment:
- [ ] In the `footer` section, the social media icons are not well styled as in the Figma design.Currently, they are to far-spaced so kindly ensure that you style them accordingly as in the Figma design :100: 

| Your Design | Figma Design |
|--------|--------|
| ![image](https://user-images.githubusercontent.com/102699958/229939951-72c3741c-75cb-4bc2-9030-40bb0fb0d9a7.png) | ![image](https://user-images.githubusercontent.com/102699958/229939997-5f21b726-6199-4a3d-87cd-22ac0a0a34b0.png) |